examples/kitchen-sink-vite/app/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>remote-ui kitchen sync</title>
+    <link rel="icon" href="https://fav.farm/🛰️" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <main>
+      <h1>remote-ui kitchen sink</h1>
+      <ul>
+        <li>
+          <a href="/vanilla-iframe/">Vanilla iframe</a>
```

Comment:
Can we add a little extra styling to make these list items nicer? Removing the default list styles + larger tap targets would be great.